Update zenodo sample testing for FMCIB

surajpaib opened the pull request.

Commit:
Update zenodo sample testing

Commit:
Fix zenodo folder hierarchy

Commit:
Fix missing workflow ref

Commit:
Fix filename

surajpaib commented:
@LennyN95 Updated the zenodo ref

LennyN95 commented:
@surajpaib I prepared sample annotations for all workflows and added exemplary descriptions to some for you to review and extend on. Can you give me write access on your fork repo?

surajpaib commented:
Great @LennyN95! You should have access already I think?

LennyN95 committed:
Add sample annotation

- add file structure annotation for all workflows
- add example descriptions for default, from_centroids, and from_json workflow

LennyN95 commented:
I opened a new code space, and now it works. Previously, I encountered a permission error.

@surajpaib, I added the sample annotation structure for all workflows. Could you please:
-  Check the descriptions
-  Add descriptions where they are missing
-  For the workflow where the dicomseg file and the dicom files are in the same folder, include the name of the dicomseg file with an appropriate description, and perhaps two exemplary slices of the dicom image?

Commit:
Update from_centroids.yml

Commit:
Update from_json.yml

Commit:
Update from_rtstruct.yml

Commit:
Update from_slicer.yml

Commit:
Update default.yml

surajpaib commented:
@LennyN95 Looks all good to me! Added some info as you requested

LennyN95 committed:
sample annotations updated
